spec/ruby/core/string/inspect_spec.rb

```diff
@@ -502,5 +502,11 @@
         "\u{3042}".encode("EUC-JP").inspect.should == '"\\x{A4A2}"'
       end
     end
+
+    describe "and the string has both ASCII-compatible and ASCII-incompatible chars" do
```

Comment:
I could probably upstream this spec to ruby/ruby?

Reply:
All changes to specs in this repo get automatically synced both ways so that'll happen without you having to do anything.